vin/app/controllers/vin/admin_controller.rb

```diff
@@ -1,47 +0,0 @@
-class Vin::AdminController < ApplicationController
-respond_to :json
-
-  # POST /admin
-  def create
-    name = params[:name]
-    admin = Admin.create({name: name})
-    if admin
-      render json: { errors: [] }
-    else
-      render json: { errors: ["Unable to create admin"]}
-    end
-  end
-
-  # GET /admin
-  def index
-    admins = Admin.all
-
-    render json: admins
-  end
-
-  # PUT /admin/{aid}
-  def update
-    admin = Admin.find(params[:id])
-    name = params[:name]
-    admin.update_attribute(:name, name)
-
-    if admin
```

Comment:
True that, it's a typo (copyed it from the `create` method, a bad habit, i know...). Is it better to use `where...first` than just `find`?